app/models/produce.rb

```diff
@@ -0,0 +1,3 @@
+class Produce < ApplicationRecord
+  validates :name, :image, presence: true
+end
```

Comment:
Curious if we will need to add relationships to listings and/or offers?

Reply:
I don't think we do since we are passing the name (maybe later the image as well) as an argument when creating a listing/offer.